README.md

```diff
@@ -143,10 +143,7 @@ Unit tests can be executed with the Gradle `test` task:
 ```
 
 The projects also includes integration tests which startup and test 
-an Alfresco with the Alfred Telemetry extension installed. 
-
-> The integration tests make use of Xenit Alfresco docker images, which are still private _for now_. Therefore,
```

Comment:
This is still valid, as external users will not be able to run the enterprise integration tests

Reply:
Those are hidden behind an `-Penterprise` flag

Reply:
Ah, okay. My approval is still valid then :smile: